polymetis/polymetis/python/polymetis/gripper_interface.py

```diff
@@ -24,30 +25,40 @@ def __init__(self, ip_address: str = "localhost", port: int = 50052):
         self.channel = grpc.insecure_channel(f"{ip_address}:{port}")
         self.grpc_connection = polymetis_pb2_grpc.GripperServerStub(self.channel)
 
+    def _send_gripper_command(self, command, msg, blocking=True) -> None:
+        if blocking:
+            command(msg)
+        else:
+            threading.Thread(target=command, args=(msg,)).start()
```

Comment:
If `command` hangs, the server cannot die. Should we use `daemon=True` or otherwise think about cleanup during exit?